source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -8,10 +8,20 @@ body[class] {
 	--wp-local-header-offset: 57px;
 }
 
-// Parsed handbook content doesn't use image blocks, so large images are
-// missing the max-width and break out of the page container.
-.entry-content p > img {
-	max-width: 100%;
+.entry-content {
+	// Parsed handbook content doesn't use image blocks, so large images are
+	// missing the max-width and break out of the page container.
+	p > img {
+		max-width: 100%;
+	}
+
+	pre:not(.line-numbers) {
```

Comment:
I don't think it's safe to style based on `line-numbers` since that could be removed easily. 
What about using `wp-block-code`? Would that work?

Reply:
I was thinking that it would be ok to be replaced by the styles here when there's only `wp-block-code` since I can't think of or find any case where we wouldn't want line numbers for a code block at the moment. But I guess it makes sense to maintain a consistent style in case we ever decide to remove `line-numbers` some day.

Reply:
It's possible that short code blocks, maybe in comments wouldn't get line-numbers... stuff like that.